javascript/javascript1/week4/homework-js1-w4.js

```diff
@@ -0,0 +1,195 @@
+// Find Class Exercises solutions in './classExercises';
+// CodeWar Profile : https://www.codewars.com/users/YuejiaoShi
+// Run my Voice Assistant on CodeSandbox: https://codesandbox.io/p/sandbox/hopeful-fermat-hdgkwc?file=%2Fsrc%2Findex.js%3A121%2C53
+
+
+// Voice assistant
+
+let userName = "";
+let toDoList = [];
+let currentToDO = "";
+const operators = [
+  "+",
+  "-",
+  "*",
+  "/",
+  "x",
+  "add",
+  "plus",
+  "minus",
+  "subtract",
+  "multiply",
+  "divide",
+  "divided by",
+];
+
+function capitalFirstChar(strOrArr) {
+  return strOrArr.charAt(0).toUpperCase() + strOrArr.slice(1);
+}
+
+function saveName(arr) {
+  const newName = arr[arr.indexOf("is") + 1];
+  return (userName = capitalFirstChar(newName));
+}
+
+function addToDo(newToDo) {
+  toDoList.push(newToDo);
+  return (currentToDO = capitalFirstChar(newToDo));
+}
+
+function getReply(command) {
+  const commandArr = command.split(" ").map((word) => word.toLowerCase());
+  // Name
+  if (command.toLowerCase().includes("my name is")) {
+    if (userName === "") {
+      saveName(commandArr);
+      return `Nice to meet you ${userName}`;
+    } else {
+      return `We've already saved your name ${userName}`;
+    }
+  } else if (command.toLowerCase().includes("what is my name")) {
+    return `Your name is ${userName}`;
+  }
+
+  // Todo
+  else if (
+    commandArr.includes("add") &&
+    command.toLowerCase().includes("to my")
+  ) {
```

Comment:
This will change as soon as you run Prettier. Might be better to just start a new `if` condition, instead of chaining many `else if`